bitnami/fluentd/values.yaml

```diff
@@ -1209,7 +1227,24 @@ aggregator:
   ## @param aggregator.extraVolumeClaimTemplates Optionally specify extra list of additional volume claim templates for the Fluentd Aggregator pods in StatefulSet
   ##
   extraVolumeClaimTemplates: []
-
+  
```

Comment:
"Empty lines" include `##` in the rest of the file. Please add it here as well to be consistent

Reply:
In this case the line should be directly removed, `##` is used to above the value, no below